packages/website/components/Topbar.tsx

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import tokens from '@contentful/f36-tokens';
+import { Subheading, Flex } from '@contentful/f36-components';
 import { css } from 'emotion';
 import Link from 'next/link';
+import { DocSearch } from './DocSearch';
 
 export const TopbarHeight = '70px';
 
 const styles = {
   header: css`
-    display: flex;
-    background-color: ${tokens.blue700};
-    color: #fff;
+    display: grid;
+    grid-template-columns: 320px auto 320px;
```

Comment:
uhmmmm I think we will need to update this once we merge the PR's updating the page's design
because these links are aligned to the content of the page, see:

![Screenshot 2022-01-10 at 17 14 59](https://user-images.githubusercontent.com/6597467/148799628-c6e9e6d2-b4f5-4d1f-9723-9819f67bf92f.png)

This basically means we will need to implement the same grid in the topbar and in the content

Reply:
yeah. I think it's better to do it in a separate pr all together.